Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -351,7 +356,11 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
    * of the camera above the ground, tiles that are far away (close to
    * the horizon) will be culled when this flag is enabled.
    */
-  UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "Cesium|Tile Culling")
+  UPROPERTY(
+      EditAnywhere,
+      BlueprintReadWrite,
+      Category = "Cesium|Tile Culling",
+      Meta = (EditCondition = "!UseLodTransitions"))
   bool EnableFogCulling = true;
```

Comment:
Same as for frustrum culling - mention how it gets overridden by UseLodTransitions in the comment, and update its value when UseLodTransitions is enabled, if possible.